### DISCUSSION TITLE
How to update in HACS

### DISCUSSION THREAD

**Discussion by ikorpershoek** [andrew-codechimp/HA-Battery-Notes#2046] [Q&A] [answered]
I'm on BN 2.4.5 yesterday there where several integration updates. By mistake, I skipped the Battery Notes upgrade to 2.4.6. It's now in the HACS list under "pending Upgrades". I feel so stupid but how do I start an update? If I click on it it brings me to the info page of BN. In the 3 dot menu, there is no option. And under 'settings' in HA there are no updates available. Any help is welcome, or simply wait for the next update?

**Comment by andrew-codechimp** [comment, marked as answer]
Hi, this update just added some translations so you can wait if you want.
Otherwise the 3 dots should have a re-download menu option, but I've never skipped an update so maybe that's missing for some reason.
If you are using HACS 2 you can make a service/action call within the developer tools like this.

```
action: update.install
target:
  entity_id: update.battery_notes_update
data:
  version: 2.4.6
```
